.github/workflows/test.yml

```diff
@@ -10,7 +10,7 @@ env:
   # Otherwise, set variable to the commit of your branch on
   # opentelemetry-python-contrib which is compatible with these Core repo
   # changes.
-  CONTRIB_REPO_SHA: f005d90ed3bc75ee6eb7297f9e3a6b55a55b22aa
+  CONTRIB_REPO_SHA: b404de2f393aaaeca73694c37fe58fecf423a707
```

Comment:
Where is this commit from? I couldn't find this in recent main commits?

Reply:
It's in the contrib PR linked in the description

Reply:
Alright, I was thinking may be this SHA was the reason, Probably not.